sdk/typescript/src/dwallet-mpc/dkg.ts

```diff
@@ -78,8 +88,12 @@ export async function launchDKGFirstRound(c: Config) {
 			showEvents: true,
 		},
 	});
-	let sessionID = (result.events?.at(0)?.parsedJson as StartSessionEvent).session_id;
-	console.log(`Session ID: ${sessionID}`);
+	let startDKGEvent = result.events?.at(0)?.parsedJson;
+	if (!isStartDKGFirstRoundEvent(startDKGEvent)) {
+		throw new Error('invalid start DKG first round event');
+	}
+	let dwalletID = startDKGEvent.event_data.dwallet_id;
+	console.log(`dwallet ID: ${dwalletID}`);
```

Comment:
```suggestion
	console.log(`dwallet ID: ${dwalletID}`);
```
remove this, never print in exposed functions